packages/component-base/test/controller-mixin.test.js

```diff
@@ -79,4 +70,12 @@ describe('controller-mixin', () => {
       expect(controller.hostConnected.calledTwice).to.be.true;
     });
   });
+};
+
+describe('ControllerMixin + Polymer', () => {
+  runTests(definePolymer, ControllerMixin);
+});
+
+describe('ControllerMixin + Lit', () => {
+  runTests(defineLit, ControllerMixin);
```

Comment:
I think it would make sense to add a test that the mixin isn't applied at all:
```js
  it('should not apply ControllerMixin to LitElement-based classes', () => {
    class TestElement extends LitElement {}

    const mixinResult = ControllerMixin(TestElement);

    expect(mixinResult).to.equal(TestElement);
  });
```

Reply:
Thanks, done.